agent/lib/tasks/release.rake

```diff
@@ -40,6 +40,20 @@ namespace :release do
     end
   end
 
+  desc 'Build ubuntu Xenial packages'
+  task :build_ubuntu_xenial => :environment do
+    rev = ENV['REV'] || '1'
+    sh('mkdir -p build')
+    sh('rm -rf build/ubuntu_xenial/')
+    sh('cp -ar packaging/ubuntu_xenial build/')
+    sh("sed -i \"s/VERSION/#{VERSION}-#{rev}/g\" build/ubuntu_xenial/#{NAME}/DEBIAN/control")
+    sh("sed -i \"s/VERSION/#{VERSION}/g\" build/ubuntu_xenial/#{NAME}/DEBIAN/postinst")
```

Comment:
The `postinst` doesn't use VERSION? should this be for the `/etc/kontena-agent.env `instead?